rollup.config.ts

```diff
@@ -68,7 +67,14 @@ const nodePlugins = [
 	alias(moduleAliases),
 	resolve(),
 	json(),
-	conditionalFsEventsImport(),
+	replace({
+		delimiters: ['', ''],
+		incude: 'fsevents-handler.js',
+		preventAssignment: true, // required, likely bug in plugin or docs
+		values: {
+			"require('fsevents')": "require('../../../src/watch/fsevents-importer').getFsEvents()"
+		}
+	}),
```

Comment:
The only problem I see is (except for the typo in "include"): If nothing is replaced, we will not get notified. E.g. fsevents-handler.js may get renamed. Maybe we could embrace it like this

```suggestion
	replace({
		delimiters: ['', ''],
		include: 'node_modules/**',
		preventAssignment: true,
		values: {
			"require('fsevents')": `require(${JSON.stringify(path.resolve('src/watch/fsevents-importer'))}).getFsEvents()`
		}
	}),
```